documentation/docs/getting_started/docker.md

```diff
@@ -2,22 +2,78 @@
 keywords:
 - documentation
 - docker
+- guide
 ---
 
-# Docker
+# Run `INX Chronicle` using Docker
 
-## Usage
+This guide describes the necessary steps to set up and run `Chronicle` as an INX plugin of a `Hornet` node to persist Tangle data, gather time series analytics using `InfluxDB`, and display it in meaningful ways using `Grafana`.
 
-The easiest way to start Chronicle is by using our supplied Dockerfile.
+## Prerequisites
 
-First you need to setup the correct permissions for the node's database:
+1. A recent release of Docker enterprise or community edition. You can find installation instructions in the [official Docker documentation](https://docs.docker.com/engine/install/).
+2. [Docker Compose V2](https://docs.docker.com/compose/install/).
+
+## Preparation
+
+Create the necessary directories for all `Hornet` and `Chronicle` databases:
+
+```
+./docker/create_dirs.sh
+```
+
+## Configuration
+
+Configure the docker images via command-line arguments by editing the `docker-compose.yml` file. 
+
+Refer to the Chronicle CLI (run `inx-chronicle --help`) for details about configuring Chronicle via command-line arguments.
+
+See the [Hornet Wiki](https://wiki.iota.org/hornet/references/configuration/) for details about configuring `Hornet`.
+
+## Environment
+
+Chronicle's docker setup uses several environment variables. These must be configured in order to run the docker compose file as-is:
+
+* `MONGODB_USERNAME`
+* `MONGODB_PASSWORD`
+* `MONGODB_CONN_STR`
+* `INFLUXDB_URL`
+* `INFLUXDB_USERNAME`
+* `INFLUXDB_PASSWORD`
+* `JWT_PASSWORD`
+* `JWT_SALT`
+
+These variables can be defined in a `.env` file and specified to docker using the `--env-file` flag.
 
 ```sh
-./docker/prepare_docker.sh
+docker compose -f docker/docker-compose.yml --env-file .env up
```

Comment:
you don't even need to do that with the `--env-file`. It's automatically read by Docker if it finds such a file in the `docker/` folder.

Reply:
Interesting. Still, better to be explicit.